readthedocs/projects/models.py

```diff
@@ -185,6 +185,9 @@ class AddonsConfig(TimeStampedModel):
     # EthicalAds
     ethicalads_enabled = models.BooleanField(default=True)
 
+    # File Tree Diff
+    filetreediff_enabled = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
I like the "generic name" `Files changed` to involve both features: docdiff and file tree diff. Due to each addon being very specific (but related, tho), I'm not able to find better names for each of them. Both names are not ideal for marketing purposes; but are pretty descriptive of what they are... for a developer at least 😅 

I think it's hard to explain what they do in a simple and nice name:

- Documentation page changes
- Files changed between versions

Reply:
@agjohnson @stsewd do you have any idea/suggestion?

Reply:
Yeah, I've also been advocating for not using `file tree diff` to describe this addon or the feature. I think it makes the most sense for both of these features, and the addons and UI, to be combined.

`Files changed` at least describes the addon now. I don't feel it makes sense as a standalone addon that users can configure though.

Before we ship this, it feels like these should be a single, unified addon/feature. I don't particular like `DocDiff` for this. `Diff view` is short and clear, `Documentation diff` works too. Something closer to that works best I think.

Reply:
We could use `visualdiff` for docdiff and `fileschanged` for filetreediff. Those names are pretty explicit about that are and I think they work well for marketing as well. What do you think?

Reply:
He, we are already using Visual Diff in the new dashboard actually:

![Screenshot_2024-11-26_18-38-46](https://github.com/user-attachments/assets/8d70ebc9-9f5d-473e-8df0-a2fe3be34cf5)

Reply:
I think visual diff covers both well, I don't see a long term reason to use two names/addons/UIs.

Short term, I don't have opinions about not using `filechanged` but I would not surface this name in configuration/UI either.

Reply:
OK. I will keep `docdiff` and `filetreediff` for the models and code names because we are already using those names in our code. When we expose this to users, I suppose we will use "Visual diff" as we are already doing, and _maybe_ "Files changed" if we have 2 different features.

I don't want to rename the model fields to use `fileschanged` if we are already using `filetreediff` for the API response, for the function that calculate the diff, etc. It will just add more confusion to the code.

I'm happy to rename the code methods, functions, model fields, if we finally find a name that we like and we convinced about 😄